### ISSUE TITLE
use convert_caffe_model.py to convert c3d_finetune_on_ucf model

### ISSUE THREAD

**Issue by Gus-Guo** [axon-research/c3d-keras#9]
I just use the convert_caffe_model.py to convert the c3d model finetuned on UCF101, I have changed the the last fc layer of c3d.py, is that all right? After the conversion, I found the accuracy is very low...I am not sure if it's right to do so..Thank you !

**Comment by chuckcho** [comment]
What change did you make on the last `fc` layer? If you changed a layer (weights/biases randomly initialized), you need to finetune your model.

**Comment by Gus-Guo** [comment]
The model I convert is the model already finetuned on the UCF101 by the author of C3D, Tran. After convert the model, I found that the accuracy is very low. Maybe that is because I change the way I load the training image..


--- 原始邮件 ---

发件人： "Chuck Cho" <notifications@github.com>
已发： 2017年4月27日 22:56
收件人： "axon-research/c3d-keras" <c3d-keras@noreply.github.com>
抄送： "Gus-Guo" <gus_guo@outlook.com>, "Author" <author@noreply.github.com>
主题： Re: [axon-research/c3d-keras] use convert_caffe_model.py to convert c3d_finetune_on_ucf model (#9)



What change did you make on the last fc layer? If you changed a layer (weights/biases randomly initialized), you need to finetune your model.

―

**Comment by chuckcho** [comment]
Were you able to get this issue fixed?

**Comment by chuckcho** [comment]
Closing as this is outside the scope of this repo.
